knowledge_base_roadmap.md

```diff
@@ -0,0 +1,92 @@
+# Knowledge base Roadmap
+
+## Ign gazebo UI
+already there: [Understanding the GUI](https://ignitionrobotics.org/docs/citadel/gui)
+Goal: introduction to the user interface of ign gazebo
+Concepts covered
+
+* Moving and rotating entities
+* Grid setting
+* Load and save configurations
+* Components inspector
+* Entity tree
+
+## Build a robot
+
+Goal: building a robot using SDF
+Concepts covered
+
+* Create your own robot using SDF file
+* Moving joints using GUI
+Quick summary:
+* Build two wheeled robot
+* download the empty world
+* include the robot in the world
+
+## Moving the robot
+
+Goal: controlling the robot using already built plugins (diff_drive)
+Concepts covered:
+
+* What is a plugin
+* How to use it
+* Differential robot control
+
+## Build an environment
+
+Goal: build environment for simulating your robot in
+Concepts covered
+
+* 2D sketch to environment
+* Customize an environment
+* Add models to the environment
+Quick summary
+* Import and edit an environment
+* Using primitive shapes(drag and drop) as models
+* Save your world and upload it to to Fuel(maybe in a separate tutorial)
+
+## Adding sensors
+
+Goal: introduction to sensors and how they can be used.
+Concepts covered
+
+* Using an already built sensors
+* Modify the sensors(shape or functionality)
+* Build your own sensor (separate tutorial)
```

Comment:
Depending on what you mean by this, I think that's not supported in Ignition yet. See this issue: https://github.com/ignitionrobotics/ign-sensors/issues/9. I think we could remove this line.